blockchain.md

```diff
@@ -1 +0,0 @@
-test C
```

Comment:
It looks like this PR is blowing away all the original specification files that I made.  We've discussed this before so I think this is just an oversight, but for review I'd like these files to be kept and integrated into the work by linking, or removed one by one as their contents get incorporated into similar pages.

Reply:
That's fair.  I'll re-add all of the ones that we don't have integrated content for.